src/components/Search/Search.jsx

```diff
@@ -78,6 +87,7 @@ class Search extends Component<Props, State> {
             placeholder={placeholder}
             type="text"
             value={this.state.text}
+            ref={input => (focused ? input && input.focus() : null)}
```

Comment:
Probably not the best solution. This will make it focus every time it renders.

Consider using the native [`autofocus`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#autofocus) instead.